### ISSUE TITLE
 model.predict([np.array([indices]), np.array([segments]), masks])[1]

### ISSUE THREAD

**Issue by LyJss** [CyberZHG/keras-bert#140]
运行的时候，智能传入两个arrays，这是什么问题，不能传入 masks？

**Comment by CyberZHG** [comment]
https://www.tensorflow.org/beta/guide/keras/masking_and_padding
